### DISCUSSION TITLE
Protein quantification with DIA NN

### DISCUSSION THREAD

**Discussion by DianaSamodova** [vdemichev/DiaNN#613] [Q&A]
Dear Vadim,
I was wondering, whether there is a command line-based option for changing protein quantification settings in DIA NN? As far as I understand, it is using proteotypic peptides for protein quantification at the moment – is it possible to specify Top3 peptides instead? 
Thanks in advance.
Best regards,
Diana

**Comment by educhicano** [comment]
Hi Diana, 
in command mode you can use the following to do it: --top 3
Best regards,
Edu

**Comment by DianaSamodova** [comment]
Hi Edu,
Thanks a lot!
Best,
Diana

**Reply by vdemichev** [reply]
PG.Normalised and PG.Quantity use all peptides, not just proteotypic. Yes, can configure the number of top peptides used for those with --top [N]. But MaxLFQ quantities are typically the best.

**Comment by johnpleid** [comment]
if I select Top3 a protein identified with 2 peptides still will be quantified with these two peptides or will appear as N/A /

**Reply by vdemichev** [reply]
Will be quantified
